logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/MessageMapper.kt

```diff
@@ -444,6 +448,7 @@ class MessageMapperImpl(
         is MessageEntityContent.ConversationMessageTimerChanged -> MessageContent.ConversationMessageTimerChanged(messageTimer)
         is MessageEntityContent.ConversationCreated -> MessageContent.ConversationCreated
         is MessageEntityContent.MLSWrongEpochWarning -> MessageContent.MLSWrongEpochWarning
+        is MessageEntityContent.VerificationDegraded -> MessageContent.VerificationDegraded(conversationMapper.fromDaoModel(protocol))
```

Comment:
to be more consistent like 'ConversationCreated' I suggest:
```suggestion
        is MessageEntityContent.ConversationDegraded -> MessageContent.VerificationDegraded(conversationMapper.fromDaoModel(protocol))
```